acceptance/tests/sameness/sameness_test.go

```diff
@@ -609,16 +608,132 @@ type cluster struct {
 	pqName         *string
 	dnsIP          *string
 	acceptors      []string
+	primaryCluster *cluster
 }
 
-func (c cluster) fullTextPartition() string {
+func (c *cluster) fullTextPartition() string {
 	if c.partition == "" {
 		return "default"
 	} else {
 		return c.partition
 	}
 }
 
+// serviceTargetCheck verifies that curling the `static-server` using the `static-client` responds with the expected
+// cluster name. Each static-server responds with a unique name so that we can verify failover occured as expected.
+func (c *cluster) serviceTargetCheck(t *testing.T, expectedName string, curlAddress string) {
```

Comment:
A lot of these functions are unchanged, I mostly refactored to use receivers instead of passing in multiple values + a cluster